src/assets/fonticons/demo-files/demo.js

```diff
@@ -1,30 +1,32 @@
-if (!('boxShadow' in document.body.style)) {
-    document.body.setAttribute('class', 'noBoxShadow');
+if (!("boxShadow" in document.body.style)) {
```

Comment:
Isn't filename should be .tsx?

Reply:
Its an generated icon, I'll fix it in next sprint because i need to test it as well and right now code is not functional